config/site_settings.yml

```diff
@@ -139,6 +139,13 @@ basic:
     default: 365
     min: 7
     max: 36500
+  ga_version:
+    type: enum
+    default: v3_analytics
+    choices:
+      - v3_analytics
+      - v4_gtag
+    client: true
```

Comment:
Is this actually used anywhere in the client javascript? As far as I can see it's just used in the ruby/erb code, so we should be able to remove the `client: true` line.